feroxbuster: add page (#9544)
tldr-pages/tldr: pages/common/feroxbuster.md

# feroxbuster

> > Brute-forces hidden paths on web servers and more.

Reviewer: ```suggestion
> Simple, fast, recursive content discovery tool written in Rust.
> Used to brute-force hidden paths on web servers and more.
```
Suggested change: I think this would be better.

> > More information: <https://github.com/epi052/feroxbuster>.

Reviewer: ```suggestion
> More information: <https://epi052.github.io/feroxbuster-docs/docs/>.
```
Optional change: How about linking to docs directly?

> - Discover specific directories and files that match in the wordlist with extensions and 100 threads and a random user agent:

Reviewer: ```suggestion
- Discover specific directories and files that match in the wordlist with extensions and 100 threads and a random user-agent:
```

> - Discover directories and files that match in the wordlist with e[x]tensions and 100 [t]hreads and a random user [A]gent:
> 
> `feroxbuster -u {{https://example.com/}} -w {{path/to/file}} -t 100 -x php,txt -A`

Reviewer 2: ```suggestion
- Discover specific directories and files that match in the wordlist with extensions and 100 threads and a random user agent:

`feroxbuster --url "{{https://example.com/}}" --wordlist {{path/to/file}} --threads {{100}} --extensions "{{php,txt}}" --random-agent`
```

- Optional change: `specific` may sound better.
- Optional change: mnemonics are redundant for long options.
- Optional change: long options available and can be used as they are easily memorable.
- Optional change: url and extensions can be quoted to protect strings from unwanted shell expansions.
- Optional change: placeholders for some options can be used.

> `feroxbuster --url "{{https://example.com/}}" --wordlist {{path/to/file}} --threads {{100}} --extensions "{{php,txt}}" --random-agent`

Reviewer: ```suggestion
`feroxbuster --url "{{https://example.com}}" --wordlist {{path/to/file}} --threads {{100}} --extensions "{{php,txt}}" --random-agent`
```

> - Enumerate directories without recursion through a proxy:
> 
> `feroxbuster -u {{https://example.com/}} -w {{path/to/file}} --no-recursion --proxy http://127.0.0.1:8080`

Reviewer 2: ```suggestion
- Enumerate directories without recursion through a specific proxy:

`feroxbuster --url "{{https://example.com/}}" --wordlist {{path/to/file}} --no-recursion --proxy "{{http://127.0.0.1:8080}}"`
```

The same issue as above.

> `feroxbuster --url "{{https://example.com/}}" --wordlist {{path/to/file}} --no-recursion --proxy "{{http://127.0.0.1:8080}}"`

Reviewer: ```suggestion
`feroxbuster --url "{{https://example.com}}" --wordlist {{path/to/file}} --no-recursion --proxy "{{http://127.0.0.1:8080}}"`
```

> - Find links in webpages:
> 
> `feroxbuster -u {{https://example.com/}} --extract-links`

Reviewer 2: ```suggestion
- Find links in webpages:

`feroxbuster --url "{{https://example.com/}}" --extract-links`
```

The same issue as above.

> `feroxbuster --url "{{https://example.com/}}" --extract-links`

Reviewer: ```suggestion
`feroxbuster --url "{{https://example.com}}" --extract-links`
```

> - Filter by status code and number of chars:
> 
> `feroxbuster -u {{https://example.com/}} --filter-status 301 -S 4092`

Reviewer 2: ```suggestion
- Filter by a specific status code and number of chars:

`feroxbuster --url "{{https://example.com/}}" --filter-status {{301}} --filter-size {{4092}}`
```

The same issue as above.

> - Filter by a specific status code and number of chars:

Reviewer: ```suggestion
- Filter by a specific status code and a number of chars:
```

> `feroxbuster --url "{{https://example.com/}}" --filter-status {{301}} --filter-size {{4092}}`

Reviewer: ```suggestion
`feroxbuster --url "{{https://example.com}}" --filter-status {{301}} --filter-size {{4092}}`
```
